charts/astronomer/templates/registry/registry-statefulset.yaml

```diff
@@ -128,13 +130,15 @@ spec:
             items:
               - key: config.yml
                 path: config.yml
+        {{- if not .Values.registry.enableInsecureAuth }}
         - name: certificate
           secret:
             secretName: {{ .Values.global.tlsSecret }}
         {{ include "custom_ca_volumes" . | indent 8  }}
         {{- if and .Values.registry.gcs.enabled .Values.registry.gcs.useKeyfile }}
         {{- include "registry.gcsVolume" . | indent 8 }}
         {{- end }}
+        {{- end }}
```

Comment:
on Line no 89 - 92 you have added logic to remove only ssl certs but here in this condition you are removing everything

Reply:
corrected.